pygyro/tools/getSlice.py

```diff
@@ -1,33 +1,34 @@
-from mpi4py                 import MPI
+from pygyro.initialisation.setups import setupFromFile
+import h5py
+from mpi4py import MPI
 import time
 setup_time_start = time.clock()
 
-import h5py
-#~ import cProfile, pstats, io
+# ~ import cProfile, pstats, io
```

Comment:
Maybe we can remove the tildes? They were inserted by my old editor which commented with `#~` for some reason

Reply:
done